content/docs/SUSHI/running/_index.md

```diff
@@ -50,7 +51,39 @@ While SUSHI is running, it will print status messages as it processes your proje
 ╚═════════════════════════════════════════════════════════════════╝
 ```
 
-#### Error Messages
+### Initializing SUSHI Project
```

Comment:
I feel like maybe this would be better in the `Project Structure` and/or `Configuration` sections -- as it's really just a jump-start on those things...  If a user is working through the documentation in order, they'll probably be disappointed if they get to the Running SUSHI section and realize they manually did what SUSHI could have done for them.

I also think the doc just flows better if `Running SUSHI` is really just about running SUSHI _on a project_.  Would you agree or would you like to discuss this further?

Reply:
Also... I think `Initializing a SUSHI Project` reads better (with the `a`).

Reply:
I debated where to put it anyway, so if you think it works better in `Project Structure`, that's a-ok with me, I'll move it.

Reply:
Alright @cmoesel, https://github.com/FSHSchool/site/pull/9/commits/bc6a16774333fe42a07d32b781ef8c482b5b33b4 moves the `--init` documentation to `Project Structure`.